sdk/tables/Azure.Data.Tables/src/TableBatchResponse.cs

```diff
@@ -32,12 +30,12 @@ internal TableBatchResponse(ConcurrentDictionary<string, (HttpMessage Message, R
         /// <returns></returns>
```

Comment:
nit:  Since this is a public member, should we consider adding content for `<returns>` or removing the tag?